chain/x/zoracle/internal/keeper/request.go

```diff
@@ -97,6 +97,28 @@ func (k Keeper) ValidateDataSourceCount(ctx sdk.Context, id types.RequestID) sdk
 	return nil
 }
 
+// PayDataSourcesFee send fees to the owner of requested data sources.
+func (k Keeper) PayDataSourcesFee(ctx sdk.Context, id types.RequestID, sender sdk.AccAddress) sdk.Error {
```

Comment:
Can you add a simple test for this function? By set a request. data source and raw data request and make sure the balance of requester is reduced.